rootconf/default/etc/init.d/dropbear.sh

```diff
@@ -14,7 +14,7 @@ daemon=/usr/sbin/$name
 
 [ -x $daemon ] || exit 0
 
-ARGS="-m -B -P"
+ARGS="-m -B"
```

Comment:
`-P` suppose to have valid pid filename previous version seems accept it empty but new one won't start with empty and by default it will use `/var/run/dropbear.pid` if not `-P` is supplied.